pytket/extensions/qiskit/backends/aer.py

```diff
@@ -300,10 +300,13 @@ def process_circuits(
                     c0, ppcirc_rep = tkc, None
 
                 qc = tk_to_qiskit(c0, replace_implicit_swaps)
-                if self.supports_state:
+
+                if self.supports_state or self.supports_density_matrix:
                     qc.save_state()
```

Comment:
Shouldn't we be using https://qiskit.github.io/qiskit-aer/stubs/qiskit_aer.library.save_density_matrix.html ?

Reply:
Yes, the `QuantumCircuit.save_state` gave equivalent answers but I think you're right. Its more correct to save the density matrix.

Thanks